bin/templates/project/app/build.gradle

```diff
@@ -117,7 +117,7 @@ if (ext.cdvReleaseSigningPropertiesFile == null && file('../release-signing.prop
 // Cast to appropriate types.
 ext.cdvBuildMultipleApks = cdvBuildMultipleApks == null ? false : cdvBuildMultipleApks.toBoolean();
 ext.cdvVersionCodeForceAbiDigit = cdvVersionCodeForceAbiDigit == null ? false : cdvVersionCodeForceAbiDigit.toBoolean();
-ext.cdvMinSdkVersion = cdvMinSdkVersion == null ? defaultMinSdkVersion : cdvMinSdkVersion
+ext.cdvMinSdkVersion = cdvMinSdkVersion == null ? defaultMinSdkVersion : Integer.parseInt('' + cdvMinSdkVersion)
```

Comment:
why the empty string added?

Reply:
Some kind of recasting. This is code that was done in bf327f391614894f84448b84bcaede613ef0d5c7 but got lost somehow.

Reply:
To be cast the value to a string if it was an integer before

Reply:
Ah, `Integer.parseInt` won't like to get an Int already?